CMakeLists.txt

```diff
@@ -481,14 +481,17 @@ if (OPENMP_FOUND)
   add_definitions(-DHAS_OPENMP)
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
+  if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "AppleClang")
+    set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /usr/local/Cellar/llvm/9.0.1/lib/libomp.dylib")
```

Comment:
So, this line is required when building with Xcode. libomp.dylib has to be linked. In hindsight, it's too specific with the version of LLVM, maybe you could suggest a better way to do this?

Reply:
Unfortunately, I don't know since I've never used OSX. I think the cmake configuration file shouldn't depend on a particular system.

I think you can solve the issue by means of cmake variables. Try to modify the compiler flags or the environment:
```bash
# I am not quite sure which variable you need.
cmake -D CMAKE_SHARED_LINKER_FLAGS=/usr/local/Cellar/llvm/9.0.1/lib/libomp.dylib path/to/sources
# Probably cmake scan the standard variables.
LDFLAGS=/usr/local/Cellar/llvm/9.0.1/lib/libomp.dylib cmake path/to/sources
```